core-rust/mesh-api-server/src/mesh_api/conversions/hashes.rs

```diff
@@ -0,0 +1,28 @@
+use crate::prelude::*;
+
+pub fn extract_transaction_intent_hash(
```

Comment:
SUPER MINOR: Perhaps this should live in `block.rs` or `transaction.rs`?

Reply:
Will do